ui-core/src/types/study.ts

```diff
@@ -10,8 +10,8 @@ export type StudyEnvironment = {
   id: string
   environmentName: string
   studyEnvironmentConfig: StudyEnvironmentConfig
-  preEnrollSurvey: Survey
-  preEnrollSurveyId: string
+  preEnrollSurvey: Survey | null
+  preEnrollSurveyId: string | null
```

Comment:
I'm not certain, but I think these keys are not present if there's no pre-enroll survey (assuming these behave the same way as portal environment's pre-registration surveys).

```suggestion
  preEnrollSurvey?: Survey
  preEnrollSurveyId?: string
```